### DISCUSSION TITLE
(xcrun missing) : error: invalid active developer path (/Library/Developer/CommandLineTools)

### DISCUSSION THREAD

**Discussion by nachumk** [Homebrew/discussions#818] [Everyday usage]
MacOS 11.2.1
Xcode 14.4
CLT is installed, and reinstalled via manual download, sudo xcode-select --reset, xcode-select --install also tried before final reinstall.
xcrun can't be found when running brew upgrade which rebuilds freecad. Does Xcode 14.4 put xcrun in the same place? It appears to now be at /usr/bin/xcrun. I was able to complete brew upgrade by making a soft link for xcrun.
This is the error I get:
xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun

**Comment by SMillerDev** [comment]
This is not about advanced brew usage, I changed the category.

If you're using the freecad formula you'll have to contact the developers there because we can't change how they build.
